roles/jboss_brms/tasks/main.yml

```diff
@@ -61,7 +61,7 @@
   when: 'jboss_brms'
 
 - name: look for kie-api files inside candidate directories
-  raw: ls -1 {{ item }}/WEB-INF/lib/kie-api*
+  raw: ls -1 {{ item }}/WEB-INF/lib/kie-api* 2>/dev/null
```

Comment:
Most of the other items have quotes around the "{{item}}", you think you need that here?

Reply:
I'm gonna update this. It seems to work, but I prefer it to be consistent.